lbz/authz.py

```diff
@@ -43,6 +43,9 @@ def __init__(
         self.allow: dict = {}
         self.deny: dict = {}
         self._set_policy(auth_jwt, policy_override)
+        self.error = PermissionDenied(
+            log_warn=f"You don't have permission to {self.permission} on {self.resource}"
```

Comment:
I don't think it should be marked as a warning 🤔  The situations that the user is not privileged enough to get access somewhere are very common and a large number of logs of this level can make it difficult to find the more relevant ones.

I can agree with logging it as a debug, just to have information about that in logs 😉

Reply:
adjusted. this logging make debugging any auth issues much simpler

Reply:
I think the same but I was only worried about the logging level 😉